.eslintrc.js

```diff
@@ -24,14 +24,6 @@ module.exports = {
   rules: {
     '@typescript-eslint/ban-ts-ignore': OFF,
     '@typescript-eslint/explicit-module-boundary-types': OFF,
-    '@typescript-eslint/no-explicit-any': WARN,
-    'sort-keys': [
```

Comment:
Is there any part of the code you ran into a linting issue with? You can always inline `// @ts-ignore`  to ignore it in a block.

Reply:
I was getting this linting error repeatedly while developing and felt it only made code less legible.. For example `{ width,  height, mode }` would need to become `{ height, mode,  width }`. Not only ungrouping the `width` and `height` properties but also switching their order..